apps/signing/tests/test_views.py

```diff
@@ -0,0 +1,130 @@
+import os
+from datetime import datetime
+
+from django.core.urlresolvers import reverse
+
+from rest_framework.test import APITestCase
+
+from api.tests.test_jwt_auth import JWTAuthTester
+from signing.views import UploadVersionView
+from users.models import UserProfile
+from versions.models import Version
+
+
+class BaseUploadVersionCase(APITestCase, JWTAuthTester):
+    fixtures = ['base/addon_3615']
+
+    def setUp(self):
+        self.user = UserProfile.objects.get(email='del@icio.us')
+        self.api_key = self.create_api_key(self.user, 'foo')
+        self.guid = '{2fa4ed95-0317-4c6a-a74c-5f3e3912c1f9}'
+        self.view = UploadVersionView.as_view()
+
+    def url(self, guid, version):
+        return reverse('signing.version', args=[guid, version])
+
+    def create_version(self, version):
+        self.put(self.url(self.guid, version), version)
+
+    def authorization(self):
+        token = self.create_auth_token(self.api_key.user, self.api_key.key,
+                                       self.api_key.secret)
+        return 'JWT {}'.format(token)
+
+    def xpi_filepath(self, version):
+        return os.path.join(
+            'apps/signing/fixtures',
```

Comment:
might as well do `'apps', 'signing', 'fixtures', ...` since you're already using join. Our lone Windows developer, @eviljeff, will be happy :)